functions/db/deliveryRequests/onCreate.f.js

```diff
@@ -0,0 +1,134 @@
+const functions = require('firebase-functions');
+const moment = require('moment');
+const enums = require('../../Enums.js');
+
+/*
+ * When a delivery request is created, send notifications to 
+ * receiving agencies:
+ *     If one specific RA is requested, send notification regardless 
+ *     of availability.
+ *     Otherwise, send notifications to all available RAs in the
+ *     pending list.
+ */
+exports = module.exports = functions.database
+    .ref('/delivery_requests/{pushId}')
+    .onCreate(event => {
+        // TODO: setup Admin SDK in the future?
+        const requestRef = event.data.ref;
+        const accountsRef = requestRef.parent.parent.child('accounts');
```

Comment:
Can we make these absolute paths? `functions.database.ref('/accounts/')`

Reply:
afaik, we'll need to use either 

1. The `ref` from `DeltaSnapShot` (`event` in here) given, or 
2. Set up Admin SDK for firebase if we want to use absolute path, and do something like: `admin.database().ref('accounts')`

since the server running this code isn't technically "logged in" to read/write data. I would very much like to use absolute path (hence the `TODO` comment), but I was too lazy and didn't want to spend a lot of time figuring out how to set it up when this worked.. :p

------------------------

I just tried `functions.database.ref('accounts')` and it didn't work. I think it's a completely different kind of reference:
https://firebase.google.com/docs/reference/functions/functions.database.RefBuilder
It only has the 4 trigger functions.